### ISSUE TITLE
Medley-solos-DB

### ISSUE THREAD

**Issue by lostanlen** [mir-dataset-loaders/mirdata#62]
Medley-solos-DB is a cross-collection dataset for automatic musical instrument recognition in solo recordings. It consists of a training set of 3-second audio clips, which are extracted from the MedleyDB dataset of Bittner et al. (ISMIR 2014) as well as a test set of 3-second clips, which are extracted from the solosDB dataset of Essid et al. (IEEE TASLP 2009). Each of these clips contains a single instrument among a taxonomy of eight: clarinet, distorted electric guitar, female singer, flute, piano, tenor saxophone, trumpet, and violin.

The Medley-solos-DB dataset is the dataset that is used in the benchmarks of musical instrument recognition in the publications of Lostanlen and Cella (ISMIR 2016) and Andén et al. (IEEE TSP 2019).

https://ieeexplore.ieee.org/abstract/document/8721532/

Medley-solos-DB is available on Zenodo: https://zenodo.org/record/2582103 

I am personally working on a PR. My branch is https://github.com/lostanlen/mirdata/tree/medley-solos

**Comment by lostanlen** [comment]
That's good to know. Thank you for the reminder. I will prepare a PR asap.
Is there a planned date for beta release?
